eisbuk-admin/src/components/atoms/AttendanceSheet/AttendanceSheet.stories.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import AttendanceSheetSlot from "./AttendanceSheetSlot";
+import {
+  baseProps,
+  saul,
+  walt,
+  intervalStrings as intervals,
+} from "@/components/atoms/AttendanceCard/__testData__/dummyData";
+export default {
+  title: "AttendanceSheetSlot",
```

Comment:
You can name the titles of stories with whitespace: "Attendance Sheet Slot", this way it's a bit prettier in storybook.